manager/integration/tests/test_upgrade.py

```
@@ -733,7 +620,13 @@ def test_upgrade(longhorn_install_method,
             print(f"Detached volume {v.name}")
 
     engineimages = client.list_engine_image()
+    # the longhorn engine image to be upgraded to
+    # it should be defined, so os.environ is used to throw error if it's not found
+    longhorn_engine_image = os.environ['CUSTOM_LONGHORN_ENGINE_IMAGE']
+    print(f"target longhorn engine image = {longhorn_engine_image}")
+    print(f"listing available longhorn engine images:")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)